play-validations/memory-footprint/src/main/java/com/google/wear/watchface/dfx/memory/ResourceMemoryEvaluator.java

```diff
@@ -112,11 +112,25 @@ private static void evaluateInHumanReadableMode(EvaluationSettings settings) {
      */
     static List<MemoryFootprint> evaluateMemoryFootprint(EvaluationSettings evaluationSettings) {
         try (InputPackage inputPackage = InputPackage.open(evaluationSettings.getWatchFacePath())) {
+
+            AndroidManifest manifest = inputPackage.getManifest();
```

Comment:
Move this to line 122, so that it executes only if the `--schema-version` is not homeyface. If it is honeyface, then we won't throw in case the <property> is not defined.